src/peripheral/scb.rs

```diff
@@ -640,4 +643,29 @@ impl SCB {
             (*Self::ptr()).icsr.write(SCB_ICSR_PENDSVCLR);
         }
     }
+
+    /// Set the PENDSTCLR bit in the ICSR register which will clear a pending SysTick interrupt
+    #[inline]
+    pub fn set_systick(&mut self) {
```

Comment:
I'm not sure about calling this `set_systick` when just above we have `set_pendsv`. Really this one should be `set_pendst` or we need to rename both (causing a major version change). Same applies to `is_systick_pending` and `clear_systick_pending`. I'd prefer just using `pendst` here.

Reply:
Yes, `set_pendst` looks better.